utils/src/main/java/com/cloud/utils/NumbersUtil.java

```diff
@@ -93,6 +95,13 @@ public static String toReadableSize(long bytes) {
         return builder.toString();
     }
 
+    public static String toHumanReadableSize(long size) {
```

Comment:
@Spaceman1984 can you add unit test for this method, giving some values such as zero and non-zero (to test KB, MB, GB, TB). Have you verified that all consumes would pass `long` to it and not `Long` which may potentially cause a NPE?

Reply:
I have added a check and a test for null which prevents an NPE.